ddl/ddl_api.go

```diff
@@ -255,6 +255,10 @@ func checkColumnCantHaveDefaultValue(col *table.Column, value interface{}) (err
 	return nil
 }
 
+func isExplicitTimeStamp() bool {
+	return strings.EqualFold(variable.GetSysVar("explicit_defaults_for_timestamp").Value, "ON")
```

Comment:
GetSysVar will get the default value of the system variable.

Reply:
This is a None scope variable, it's read-only.

Reply:
The code here are used to clarify it should be changed in the future. Anyway, how about return true directly then add a TODO comment on it. @shenli @coocood